install.py

```diff
@@ -27,15 +34,33 @@
 from rez.vendor.distlib.scripts import ScriptMaker  # noqa: E402
 
 
-def create_virtual_environment(dest_dir):
-    builder = venv.EnvBuilder(with_pip=True)
-    builder.create(dest_dir)
+def create_virtual_environment(dest_dir) -> None:
+    """Create a virtual environment in the given directory.
+
+    Args:
+        dest_dir (str): Full path to the virtual environment directory.
 
+    """
+    if USE_VIRTUALENV:
+        try:
+            subprocess.run(["virtualenv", dest_dir], check=True)
```

Comment:
```suggestion
            subprocess.run([sys.executable, "-m", "virtualenv", dest_dir], check=True)
```
Virtualenv has to come from the current executable. We need to enforce this.

Reply:
@JeanChristopheMorinPerso Thank you for pointing that, I have added the commit.